File: OOP/2_method.cpp
#include <iostream>
#include <string>
using namespace std;

class Genz
{
public:
    // Constructor
    Genz()
    {
        cout << "hello" << endl;
    }

    // Method
    string display()
    {
        cout << "this is genz" << endl;
        return "ok";
    }
};

string display()
{
    cout << "this is another display" << endl;
    return "fine";
}

int main()
{
    Genz a;
    string result = display();
    cout << result << endl;
    return 0;
}

File: OOP/2_method.py
class Genz:
    def __init__(self) -> None:
        print('hello')
    def display(self):
        print('this is genz')
        return 'ok'

def display():
    print('this is another display')
    return 'fine'

a = Genz()
a.display = display()
print(a.display)










# class A:
#     pass

# def display():
#     print("I am Methods")

# obj = A()
# obj.value = display

# print(obj.value)
# print(obj)
